compiler-cli/src/module.rs

```diff
@@ -0,0 +1,38 @@
+use lazy_static::lazy_static;
+
+/// Check if a module name is a valid gleam module name.
+pub fn is_gleam_module(module: &str) -> bool {
```

Comment:
How come this function is exported now? I couldn't spot a new use of it. I'd prefer it to be private, if it's public I'd like it moved to core, though not entirely sure where it should go.

Reply:
reverted this change